core/domain/exp_domain.py

```diff
@@ -655,6 +659,75 @@ def validate(self):
         self.outcome.validate()
 
 
+class Hint(object):
+    """Value object representing a hint."""
+    def __init__(self, hint_text):
+        self.hint_text = html_cleaner.clean(hint_text)
+
+    def to_dict(self):
+        return {
+            'hint_text': self.hint_text,
+        }
+
+    @classmethod
+    def from_dict(cls, hint_dict):
+        return cls(html_cleaner.clean(hint_dict['hint_text']))
```

Comment:
Just to check, do we need to do the cleaning here, given that we do it in `__init__`?